src/callbacks.js

```diff
@@ -51,25 +51,53 @@
  */
 
 /**
+ * @private
+ * @deprecated
  * @callback pc.callbacks.LoadHierarchy
  * @description Callback used by {@link pc.Application#loadSceneHierarchy}.
  * @param {string|null} err - The error message in the case where the loading or parsing fails.
  * @param {pc.Entity} [entity] - The loaded root entity if no errors were encountered.
  */
 
 /**
+ * @private
+ * @deprecated
  * @callback pc.callbacks.LoadSettings
  * @description Callback used by {@link pc.Application#loadSceneSettings}.
  * @param {string|null} err - The error message in the case where the loading or parsing fails.
  */
 
 /**
+ * @private
+ * @deprecated
```

Comment:
Why deprecate the old callbacks? Just update the description to say the updated function names of the new API.